Performance/Harness.h

```diff
@@ -46,7 +46,9 @@ class Harness {
   void run();
 
 private:
-  /** Include fractional parts in milliseconds. */
+  /** Microseconds unit for representing times. */
+  typedef std::chrono::duration<double, std::micro> microseconds_d;
+  /** Milliseconds unit for representing times. */
   typedef std::chrono::duration<double, std::milli> milliseconds_d;
```

Comment:
Good catch!  It was still being used for the overall time summary until (at the last minute) I realized that I didn't care about the overall time, just the relative stddev.